src/goto-synthesizer/cegis_verifier.cpp

```diff
@@ -357,7 +357,7 @@ cext cegis_verifiert::build_cex(
   const source_locationt &loop_entry_loc)
 {
   // Valuations of havoced variables right after havoc instructions.
-  std::unordered_map<exprt, mp_integer, irep_hash> object_sizes;
+  std::unordered_map<exprt, bytest, irep_hash> object_sizes;
   std::unordered_map<exprt, mp_integer, irep_hash> havoced_values;
   std::unordered_map<exprt, mp_integer, irep_hash> havoced_pointer_offsets;
```

Comment:
What about the the template parameter types in `havoc_values` and `havoced_pointer_offsets`? Does it make sense to change the types from `mp_integer` to `bitst`/`bytest` as well?

Reply:
Good call, done.